backend/app/controllers/projects.js

```diff
@@ -253,9 +255,18 @@ module.exports = {
         var thunkQuery = req.thunkQuery;
         co(function* () {
             yield * checkProjectData(req);
-            var updateObj = _.pick(req.body, ['title', 'description', 'startTime', 'closeTime', 'status', 'codeName']);
+            var updateObj = _.pick(req.body, ['title', 'description', 'startTime', 'closeTime', 'status', 'codeName', 'firstActivated']);
             var result = false;
             if (Object.keys(updateObj).length) {
+
+                var project = yield thunkQuery(
+                    Project.select().where(Project.id.equals(req.params.id))
+                );
+
+                // Update firstActivated if the status was changed from 0 to 1
+                if (updateObj.status == 1 && project.firstActivated === null) {
```

Comment:
We should strict (`===`) compare the status, and `project` here is an array from the query above, so we need to access `firstActivated` from the first (and only) element